src/identifiers/models.py

```diff
@@ -21,6 +21,12 @@
     ('pubid', 'Publisher ID'),
 )
 
+IDENTIFIER_TYPES = [
+    'uri',
+    'pubid',
+    'id',
```

Comment:
This might look confusing as it doesn't include `'doi'`, why not include it here and change the order of the urls below?
[ ...]
url(<doi_url>) # more restrictive goes first.
url(identifier_url)